doc/rtd/topics/faq.rst

```diff
@@ -226,12 +226,63 @@ custom network config.
 .. _Instance Configuration: https://linuxcontainers.org/lxd/docs/master/instances
 .. _Custom Network Configuration: https://linuxcontainers.org/lxd/docs/master/cloud-init
 
+cloud-localds
+-------------
+
+The `cloud-localds` command from the cloud-utils generates a disk with user
```

Comment:
a link to the tool or the utils might be a good idea

Reply:
How about to the upstream project like: https://github.com/canonical/cloud-utils/

Reply:
Added a link!